tests/scripts/run_jacobian_validation.sh

```diff
@@ -215,4 +215,4 @@ else
 
     # Run rk_tolerance_comparision.C
     root '../../../../tests/validation/root/rk_tolerance_comparison.C+O('${log10_min_rk_tol}','${log10_max_rk_tol}')'
```

Comment:
Shouldn't this directory be changed as you moved the script to other directory

Reply:
True, but I did not rename it in the end. Let me update the description

Reply:
Oh, oops, absolutely.